<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="Panda3DContainer" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="expression_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>131</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>151</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="store_button">
      <property name="text">
       <string>store</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="reset_button">
      <property name="text">
       <string>reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="store_file">
      <property name="text">
       <string>save_file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="play_button">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="play_file">
      <property name="text">
       <string>play_file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>HEAD   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>-90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Slider1">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>LEFT      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>-90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Slider3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>BODY   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>-90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Slider2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>RIGHT   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>-90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Slider4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Slider_num1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Slider_num2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Slider_num3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="Slider_num4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 0</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>721</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="capturedImagesLayout">
    <item>
     <widget class="QScrollArea" name="capturedImagesScrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollwidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>717</width>
         <height>257</height>
        </rect>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>671</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionpage">
   <property name="text">
    <string>page</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
